paintBrush.js

```diff
@@ -133,5 +92,47 @@ function clickShow(e){
 }
 
 
+// Functions to stop and start painting, and erasing
+function stopPainting() {
+    painting = false;
+}
+
+function eraseTrue() {
+    eraseMode = true;
+}
+
+function eraseFalse() {
+    eraseMode = false;
+}
+
+function startPainting() {
+    painting = true;
+}
```

Comment:
Can you add comments above these functions as well as any others that don't have any